powerdnsadmin/routes/index.py

```diff
@@ -684,6 +700,31 @@ def register():
         return render_template('errors/404.html'), 404
 
 
+# Show welcome page on first login if otp_force is enabled
+@index_bp.route('/welcome', methods=['GET', 'POST'])
+def welcome():
+    if 'welcome' not in session:
+        return redirect(url_for('index.index'))
+
+    user_id = session['user_id']
+    user = User(id = user_id)
```

Comment:
Please change to `user = User(id=user_id)`.

Reply:
I haven't fully understood this comment.

My solution was to replace lines 709,710 with:
`user = User(id = session['welcome_user_id'])`.

Please let me know if this isn't ideal.

Reply:
Ah - my fault! Semantically it's all good, just remove the spaces around the = sign for the keyword argument. PEP 8 [recommends this](https://www.python.org/dev/peps/pep-0008/#other-recommendations) in order to visually distinguish keyword arguments from variable assignments.

Could you please also remove [this now unused import](https://github.com/PowerDNS-Admin/PowerDNS-Admin/blob/60b84162fa5207b0b48e0d193245a404358bcfdc/powerdnsadmin/routes/index.py#L15)?

Reply:
Oh ok, I get it now, thanks for the explanation. Done!